test_cosbench_workload_operations fails with FileNotFoundError

Relevant runs :

https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#ocs/launches/all/47342/2111687/2111704/log 
https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#ocs/launches/all/47346/2111882/2111899/log 
https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#ocs/launches/all/47355/2112171/2112188/log 

`self = <tests.functional.workloads.app.cosbench.test_cosbench.TestCosbenchWorkload object at 0x7f6d375de5d0>
cosbench = <ocs_ci.ocs.cosbench.Cosbench object at 0x7f6d5f7b9c50>


def test_cosbench_workload_operations(self, cosbench):
    """
    Test to perform reads and writes on objects.

    """
    bucket_prefix = "bucket-"
    buckets = 10
    objects = 50

    # Operations to perform and its ratio(%)
    operations = {"read": 50, "write": 50}

    # Deployment of cosbench
    cosbench.setup_cosbench()

    # Create initial containers and objects
    cosbench.run_init_workload(
        prefix=bucket_prefix, containers=buckets, objects=objects, validate=True
    )

    # Run main workload

  cosbench.run_main_workload(
        operation_type=operations,
        prefix=bucket_prefix,
        containers=buckets,
        objects=objects,
        validate=True,
    )

tests/functional/workloads/app/cosbench/test_cosbench.py:101: 



ocs_ci/ocs/cosbench.py:392: in run_main_workload
    throughput, bandwidth = self.get_performance_result(



self = <ocs_ci.ocs.cosbench.Cosbench object at 0x7f6d5f7b9c50>
workload_name = 'workload_name', workload_id = 'w2', size = 64


def get_performance_result(self, workload_name, workload_id, size):
    workload_file = self.get_result_csv(
        workload_id=workload_id, workload_name=workload_name
    )
    throughput_data = {}
    bandwidth_data = {}

  with open(workload_file, "r") as file:
E       FileNotFoundError: [Errno 2] No such file or directory: '/tmp/cosbench-tool-_xoz70i7/w2-workload_name.csv'



ocs_ci/ocs/cosbench.py:651: FileNotFoundError`

--- a/tests/functional/workloads/app/cosbench/test_cosbench.py
+++ b/tests/functional/workloads/app/cosbench/test_cosbench.py
@@ -104,6 +104,7 @@ class TestCosbenchWorkload(E2ETest):
             containers=buckets,
             objects=objects,
             validate=True,
+            timeout=1200,
         )
 
         # Dispose containers and objects

